site/homepage/content/getting-started-kafka/index.md

```diff
@@ -0,0 +1,440 @@
+
+HTTP/1.1 204 Updated
+etag: cf91fd4d-7111-4e8a-af68-c703993a8be1
+Content-Length: 0
+~~~
+
+<a name="starting-a-consumer"></a>
+## Starting the example Application
+
+The telemetry data produced by devices is usually consumed by downstream applications that use it to implement their corresponding business functionality.
+In this guide we will use the Hono command line client to simulate such an application.
+The client will connect to the Kafka cluster that provides Hono's north bound Kafka based [Telemetry]({{% doclink "/api/telemetry-kafka/" %}}) and [Event API]({{% doclink "/api/event-kafka/" %}})s, subscribe to all telemetry and event messages and log the messages to the console.
+
+Open a new terminal window, set the `KAFKA_IP` environment variable and start the client from the command line. If you
+are using the Sandbox server (make sure to replace `my-tenant` with your tenant identifier):
+
+~~~sh
+# in directory where the hono-cli-*-exec.jar file has been downloaded to
+export MY_TENANT=my-tenant
+export KAFKA_IP=hono.eclipseprojects.io
+java -jar hono-cli-*-exec.jar --hono.kafka.commonClientConfig.bootstrap.servers=$KAFKA_IP:9092 --hono.kafka.commonClientConfig.security.protocol=SASL_PLAINTEXT --hono.kafka.commonClientConfig.sasl.jaas.config="org.apache.kafka.common.security.scram.ScramLoginModule required username=\"hono\" password=\"hono-secret\";" --hono.kafka.commonClientConfig.sasl.mechanism=SCRAM-SHA-512 --spring.profiles.active=receiver,kafka --tenant.id=$MY_TENANT
+~~~
+
+Otherwise, if you are using a local Minikube cluster, first save the truststore file of the Kafka broker to your file
+system with the following command (adapt the environment variable `KAFKA_TRUSTSTORE_PATH` to change where the truststore will be stored):
+
+~~~sh
+export KAFKA_TRUSTSTORE_PATH=/tmp/truststore.jks
+kubectl get secrets eclipse-hono-kafka-jks --template="{{index .data \"kafka.truststore.jks\" | base64decode}}" -n hono > $KAFKA_TRUSTSTORE_PATH
+~~~
+
+and then start the client (make sure to replace `my-tenant`):
+
+~~~sh
+# in directory where the hono-cli-*-exec.jar file has been downloaded to
+export MY_TENANT=my-tenant
+export KAFKA_IP=$(kubectl get service eclipse-hono-kafka-0-external --output="jsonpath={.status.loadBalancer.ingress[0]['hostname','ip']}" -n hono)
+java -jar hono-cli-*-exec.jar --hono.kafka.commonClientConfig.bootstrap.servers=$KAFKA_IP:9092 --hono.kafka.commonClientConfig.security.protocol=SASL_SSL --hono.kafka.commonClientConfig.sasl.jaas.config="org.apache.kafka.common.security.scram.ScramLoginModule required username=\"hono\" password=\"hono-secret\";" --hono.kafka.commonClientConfig.sasl.mechanism=SCRAM-SHA-512 --spring.profiles.active=receiver,kafka --tenant.id=$MY_TENANT --hono.kafka.commonClientConfig.ssl.truststore.location=$KAFKA_TRUSTSTORE_PATH --hono.kafka.commonClientConfig.ssl.truststore.password=honotrust --hono.kafka.commonClientConfig.ssl.endpoint.identification.algorithm="" 
```

Comment:
this looks very complicated :-( Are you sure that it still reflects the command client client's current state/requirements?

Reply:
Alternatively, we could skip all the parameters for the authentication and use the sandbox profile from the CLI that already contains them as defaults. Would you prefer that?

Reply:
Wasn't that the overall idea when introducing the profile(s)?

Reply:
Please have a look at my latest commit.